### PR TITLE
Add registration page

### PR DESCRIPTION
**Pull request by Misiek22113**
image
image
image
image
image

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -15,20 +17,93 @@ const montserrat = Montserrat({
...
+          color: "#ffffff",
+          "& .MuiSvgIcon-root": {
+            fontSize: "30px",
+          },
+        },
+      },
+    },
+  },
+});
+
```
Comment (Gujansko) [inline src/app/layout.tsx:92]: I would love to see theme setup in another file.

File: src/app/participant/page.tsx [hunk 2]
```diff
@@ -13,11 +16,15 @@ export default function Participant() {
         setSelectedSection={setSelectedSection}
       />
       <SectionContent selectedSection={selectedSection}>
-        <SectionContentElement className="bg-cyan-950 text-left">
-          <span>Content 1</span>
+        <SectionContentElement className="bg-black text-left">
+          <span>
+            <RegistrationForm />
+          </span>
```
Comment (Gujansko) [inline src/app/participant/page.tsx:22]: These spans are not needed just `<RegistrationFrom />`

File: src/app/participant/page.tsx [hunk 3]
```diff
@@ -13,11 +16,15 @@ export default function Participant() {
...
         </SectionContentElement>
-        <SectionContentElement className="bg-red-900 text-center">
-          <span>Content 2</span>
+        <SectionContentElement className="bg-black text-left">
+          <span>
+            <PaymentsForm />
+          </span>
```
Comment (Gujansko) [inline src/app/participant/page.tsx:27]: Same as above

File: src/components/RegistrationForm/RegistrationForm.tsx [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+            name="additionalAccept"
+          />
+        </div>
+        <div className="flex gap-4 items-center justify-center m-8">
+          <button type="submit" className="button-round button-filled ">
+            Submit
+          </button>
+        </div>
+      </form>
+      {/* <DevTool control={methods.control} /> */}
```
Comment (Gujansko) [inline src/components/RegistrationForm/RegistrationForm.tsx:242]: What's this?

File: src/components/RegistrationForm/selectOptions.ts [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+};
+
+export const sessionOptions = {
+  PL: ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12"],
+};
+
+export const statuteConsentLabel = "Akceptuję regulamin";
+export const personalConsentLabel =
+  "Wyrażam zgodę na przetwarzanie danych osobowych";
+export const additionalConsentLabel = "Wyrażam zgodę na";
```
Comment (Gujansko) [inline src/components/RegistrationForm/selectOptions.ts:61]: Shouldn't these have engilsh versions?

File: src/app/layout.tsx [hunk 6]
```diff
@@ -15,20 +17,93 @@ const montserrat = Montserrat({
...
   return (
     <html lang="en">
-      <body
-        className={`${montserrat.className} p-0 m-0 box-border overflow-x-hidden`}
-      >
-        <LanguageModeContextProvider>
-          <Navbar />
-          {children}
-        </LanguageModeContextProvider>
+      <body className={`${montserrat.className} h-screen p-0 m-0 box-border`}>
```
Comment (Gujansko) [inline src/app/layout.tsx:100]: overflow-x-hidden disappeared

File: src/components/RegistrationForm/RegistrationForm.tsx (hunk 4 above)
Comment (Misiek22113) [reply src/components/RegistrationForm/RegistrationForm.tsx:242]: Tool that helps to see real time changes in form. I'am using it for checking if everything is fine.

File: src/components/RegistrationForm/selectOptions.ts (hunk 5 above)
Comment (Misiek22113) [reply src/components/RegistrationForm/selectOptions.ts:61]: there will be

**Review by Gujansko — APPROVED** [review]

**Pull request merged**